// @slippi/slippi-js/dist/types.esm.js
var Command;

(function (Command) {
  Command[Command["SPLIT_MESSAGE"] = 16] = "SPLIT_MESSAGE";
  Command[Command["MESSAGE_SIZES"] = 53] = "MESSAGE_SIZES";
  Command[Command["GAME_START"] = 54] = "GAME_START";
  Command[Command["PRE_FRAME_UPDATE"] = 55] = "PRE_FRAME_UPDATE";
  Command[Command["POST_FRAME_UPDATE"] = 56] = "POST_FRAME_UPDATE";
  Command[Command["GAME_END"] = 57] = "GAME_END";
  Command[Command["FRAME_START"] = 58] = "FRAME_START";
  Command[Command["ITEM_UPDATE"] = 59] = "ITEM_UPDATE";
  Command[Command["FRAME_BOOKEND"] = 60] = "FRAME_BOOKEND";
  Command[Command["GECKO_LIST"] = 61] = "GECKO_LIST";
})(Command || (Command = {}));

var GameMode;

(function (GameMode) {
  GameMode[GameMode["VS"] = 2] = "VS";
  GameMode[GameMode["ONLINE"] = 8] = "ONLINE";
})(GameMode || (GameMode = {}));

var Language;

(function (Language) {
  Language[Language["JAPANESE"] = 0] = "JAPANESE";
  Language[Language["ENGLISH"] = 1] = "ENGLISH";
})(Language || (Language = {}));

var Frames;

(function (Frames) {
  Frames[Frames["FIRST"] = -123] = "FIRST";
  Frames[Frames["FIRST_PLAYABLE"] = -39] = "FIRST_PLAYABLE";
})(Frames || (Frames = {}));

export { Command, Frames, GameMode, Language };
